docs/api/index.md

```diff
@@ -30,10 +32,15 @@ For compatibility with Jest, `TestFunction` can also be of type `(done: DoneCall
 ### test.skip
 
 - **Type:** `(name: string, fn: TestFunction, timeout?: number) => void`
+- **Alias:** `it.skip`
 
-  Use `.skip` to avoid running certain tests
+  If you want to skip running some certain tests, but you don't want to delete the code due to any reason, you can use `test.skip` to avoid running them.
```

Comment:
```suggestion
  If you want to skip running certain tests, but you don't want to delete the code due to any reason, you can use `test.skip` to avoid running them.
```